templates2/read/read.html

```diff
@@ -5,34 +5,30 @@
 <script type="text/javascript" src="https://js.live.net/v7.2/OneDrive.js"></script>
 <script type="text/javascript">
     function launchOneDrivePicker(){
-        console.log("Launching OneDrive picker");
         var odOptions = {
-            clientId: "fffa9abf-8d14-45a0-a97e-8059275810bf",
+            clientId: "{{ client_id }}",
             action: "query",
             multiSelect: true,
             advanced: {
-                redirectUri: "https://tola.test/onedrive",
-                accessToken: "{{ access_token }}"
+                redirectUri: "{{ redirect_uri }}"
             },
-            success: function(files) {
-                console.log("Success");
-                console.log(files);
-                $("#id_onedrive_file").val(files.value[0].id)
+            success: function(data) {
+                console.log(data);
+                $("#id_onedrive_access_token").val(data.accessToken);
+                $("#id_onedrive_file").val(data.value[0].id);
             },
-            cancel: function() {
-                console.log("Process cancelled")
-            },
-            error: function(e) {
-                console.log("Process failed")
-            }
+            cancel: function() {},
+            error: function(e) {}
```

Comment:
If we're not handling issues with the `cancel` and `error` functions, you can remove them.